src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Login from './components/login/login';
+import NotFound from './components/pagenotfound.js/notfound';
+import Furniture from './components/dashboard/furniture/Furniture';
+import Home from './components/dashboard/home';
+import Signup from './components/login/signup';
+import Dashboard from './components/dashboard/dashboard';
+import FurnitureForm from './components/dashboard/furniture/FurnitureForm';
+import AppointmentForm from './components/dashboard/appointment/appointmentForm';
+import AppointmentsList from './components/dashboard/appointment/appointmentsList';
+import DeleteFurniture from './components/dashboard/furniture/DeleteFurniture';
+
+function App() {
```

Comment:
- [ ] I can't logout using the button:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/4978362b-ed0a-4953-a7b9-9df401435155)

The token is removed from local storage, but not the user:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/c763e0bb-acc0-4ce8-bccc-aa69a32b2872)

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/0b136270-11c9-4d99-91ec-4425bf76dae3)

I had to manually remove the local storage to logout correctly.

Kindly fix this bug so your users can logout.